<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>454</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="musicProgressSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="downloadFrame" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QPushButton" name="downloadButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>downloadOnStartup</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="downloadProgressBar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="musicFolderLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>music folder : </string>
     </property>
    </widget>
    <widget class="QLabel" name="musicFolderRepository">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>381</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="locateMusicFolderButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Locate</string>
     </property>
    </widget>
    <widget class="QLabel" name="ytdlpLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ffpmeg path : </string>
     </property>
    </widget>
    <widget class="QLabel" name="ffpmegPath">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>381</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="locateFfpmegButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Locate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fetchPlaylistButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>fetch playlist</string>
     </property>
    </widget>
    <widget class="QComboBox" name="playlistSelectionComboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>240</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="videoNumberTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="thumbnailImageLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>thumbnailImage</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="playPauseButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QLabel" name="musicNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MusicName</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutThisProject"/>
    <addaction name="aboutQt"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="apiCredentials"/>
    <addaction name="ressourcePaths"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aboutThisProject">
   <property name="text">
    <string>About this project</string>
   </property>
  </action>
  <action name="aboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="apiCredentials">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Api credentials</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="ressourcePaths">
   <property name="text">
    <string>Ressources paths</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
